modules/profile/public/lib/commands/profiles/update/build.liquid

```diff
@@ -1,13 +1,8 @@
 {% liquid
   function tokenized_names = 'modules/profile/commands/profiles/tokenize_names', object: object
-  assign name = object.first_name | append: ' ' | append: object.last_name
```

Comment:
we shouldn't delete this line if we enforce it in check https://github.com/Platform-OS/pos-module-profile/pull/10/files#diff-8186aa535baade4e9e439bec6abd033761d290e8ae2c587e2c8bc896dbe50658R6